binaries/build.py

```diff
@@ -7,7 +7,8 @@
 REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
 sys.path.append(REPO_ROOT)
 
-STAGING_SUFFIX = "-staging"
+# Change staging suffix if required
+STAGING_SUFFIX = ""
```

Comment:
this variable can be clean up. Basically, there is no difference b/w staging and prod in build script. So  all the code related to [staging](https://github.com/pytorch/serve/blob/ab6e0359527b677c64311961f3c7c2ed82d54b42/binaries/build.py#L73) can be clean up.

Reply:
Done.